packages/ui5-library-generator/src/utils/constants.ts

```diff
@@ -0,0 +1,17 @@
+export const UI5_LIB_GEN = 'UI5_LIB_GEN';
+
+export const generatorTitle = 'SAP Fiori Reusable Library';
+export const prompts = [
+    {
+        name: 'Generate UI5 Library',
+        description: 'Create a Reusable Library for use in SAP Fiori applications'
+    }
+];
+
+export const initialAnswers = {
```

Comment:
Strange that we are setting everything to empty strings. Usually we do not do this in generators. Not sure its necessary. Undefined should suffice as you are otherwise setting values that have not been input.

Reply:
This is not for setting defaults in the inquirer. 

The issue I'm facing is, the `UI5LibraryAnswers` found in `packages/ui5-library-inquirer/src/types.ts` all had optional properties. 

The `ui5-library-writer` uses `UI5LibConfig` which has required properties e.g `libraryName` & `namespace`.

So when calling the writer with the `UI5LibraryAnswers` I get the typescript error about properties cannot be undefined. 

Is there a better approach? The only alternative I can see is to pass it something like 
```
 const ui5Lib = {
            libraryName: this.answers.libraryName || defaultLibName
.....
```

Reply:
I see. I think answers interfaces may all be undefined, setting the interface to mandatory wont make any difference to the runtime as prompts may not execute or may be skipped so you are correct to assign default values when calling the `generate` if they are required. I would have defaults for ui5libConfig, not call it answers, as only inquirers can generate answers, and apply it when creating the `ui5LibConfig` object so its clear whats happening?

Reply:
Ok thanks